cleanlab_studio/internal/api/api.py

```diff
@@ -207,19 +208,19 @@ def clean_dataset(
     api_key: str,
     dataset_id: str,
     project_name: str,
-    tasktype: str,
+    task_type: str,
     modality: str,
-    modeltype: str,
+    model_type: str,
```

Comment:
comment: I'm fine w this change for now, bc we have basically no users of this, but in the future we should make sure we have a strong reason to change user-facing APIs